test/integration/consul-container/test/gateways/http_route_test.go

```diff
@@ -256,3 +256,213 @@ func TestHTTPRouteFlattening(t *testing.T) {
 	}, checkOptions{debug: false, statusCode: service1ResponseCode, testName: "service1, v2 path with v2 hostname"})
 
 }
+
+func TestHTTPRouteParentRefChange(t *testing.T) {
+	if testing.Short() {
+		t.Skip("too slow for testing.Short")
+	}
+
```

Comment:
you'll probably want to add a t.Parallel() in here.

Reply:
```suggestion
	t.Parallel()
```